src/main/java/it/arcade/hospital/offer/OfferService.java

```diff
@@ -0,0 +1,33 @@
+package it.arcade.hospital.offer;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import static org.apache.commons.collections4.CollectionUtils.isEmpty;
+
+@Service
+public class OfferService {
+	
+	private final OfferRepository offerRepository;
+
+	@Autowired
+	public OfferService(OfferRepository offerRepository) {
+		this.offerRepository = offerRepository;
+	}
+	
+	public List<Offer> getAllOffersForDepartment(Department department) {
+		List<Offer> offers = offerRepository.findAllByDepartment(department);
+		return isEmpty(offers) ? new ArrayList<Offer>() : offers;

```

Comment:
This logic:
- check if List is empty. If it is empty, return empty list. If not, return this list...
Does it sound OK to you? Why returning different empty list if we already have empty list?  

```suggestion
		return offerRepository.findAllByDepartment(department);

```